discord/ext/commands/bot.py

```diff
@@ -624,7 +624,13 @@ def _load_from_module_spec(self, spec, key):
         else:
             self.__extensions[key] = lib
 
-    def load_extension(self, name):
+    def _resolve_name(self, name, package):
+        try:
+            name = importlib.util.resolve_name(name, package)
+        except ImportError:
+            raise errors.ExtensionNotFound(name)
+
+    def load_extension(self, name, package=None):
```

Comment:
Seems better to make `package` a keyword argument.

Reply:
Yeah not a bad idea to make them keyword only.